models/containerized_model/README.md

```diff
@@ -0,0 +1,58 @@
+# How to create containerized models?
+
+## Pre-requisites
+
+- Certifai toolkit (from [CognitiveScale website](https://www.cognitivescale.com/try-certifai/)).
+- A model (pickle) stored in a cloud storage (e.g Amazon S3).
+- A base docker image which has all the dependencies at the specific versions that were used when the model was trained.
+- Locally installed:
+    - Docker
+    - Jinja2 (`pip install -U Jinja2`)
+
+
+## Step 1 - Template generation
+
+Generate the code template for containerization of your model:
+```
+./generate.sh -i certifai-model-container:latest
+```
+
+This command should create a directory called `generated-container-model` in your current directory with the generated code.
+
+For more `generate` options:
+```
+./generate.sh --help
+```
+
+## Step 2 - Copy artifacts
+Copy the `packages` folder from inside the toolkit into the generated directory `generated-container-model`:
+
+```
+cp -r <certifai-toolkit-path>/packages generated-container-model/
```

Comment:
trivial thing but... if you change this to `cp -r <certifai-toolkit-path>/packages generated-container-model/packages` it's less prone to issues. When I first did this step I used Mac filename completion and ran cp -r <certifai-toolkit-path>/packages/ generated-container-model` which copied the contents of packages but not packages!